src/main/java/uk/gov/pay/api/model/links/directdebit/MandateLinks.java

```diff
@@ -32,7 +32,9 @@
 
     @JsonProperty(PAYMENTS)
     private final Link payments;
-    
+
+    //Hidden because it is currently unused
+    @ApiModelProperty(hidden = true)
```

Comment:
I think it's using the @ApiModelProperty on the getter instead, so move the hidden property to there i recon